gtsfm/data_association/point3d_initializer.py

```diff
@@ -55,6 +56,39 @@ class TriangulationParam(Enum):
     RANSAC_TOPK_BASELINES = 3  # deterministically choose hypotheses with largest estimate baseline
 
 
+class TriangulationOptions(NamedTuple):
+    """Options for triangulation RANSAC solver."""
+
+    reproj_error_threshold: float
+    min_track_len: int
+    mode: TriangulationParam
+
+    # RANSAC parameters
+    min_inlier_ratio: float = 0.5
+    confidence: float = 0.9999
+    dyn_num_hypotheses_multiplier: float = 3.0
```

Comment:
Hi @travisdriver, thanks for the PR. Could you add docstrings explaining each of these params, e.g. `dyn_num_hypotheses_multiplier`?

Reply:
added

Reply:
thanks, marking as resolved